WordPress/Classes/ViewRelated/System/Coordinators/MySitesCoordinator.swift

```diff
@@ -5,20 +5,25 @@ class MySitesCoordinator: NSObject {
     let mySitesSplitViewController: WPSplitViewController
     let mySitesNavigationController: UINavigationController
     let blogListViewController: BlogListViewController
+    let becomeActiveTab: () -> Void
```

Comment:
It might be good for this to also be named `onBecomeActiveTab`, otherwise it sounds a little like it's a closure that will cause this to become the active tab.

Reply:
or `tabWillBecomeActive`, since it seems to happen right before the tab switching?

Reply:
> otherwise it sounds a little like it's a closure that will cause this to become the active tab.

It actually does just that! 😄 

It requests the coordinator creator (`WPTabBarController`) to make "My Sites" the active tab, which is why I chose the name `becomeActiveTab`.

The original code (which is shown as removed in line 21) was:

```swift
WPTabBarController.sharedInstance().showMySitesTab()
```

What this block does isn't much.  It just decouples this class from `WPTabBarController`, so that in theory we could change the tab bar implementation completely without making any change to this class.  It's quite a minor change but I think it's always cleaner to avoid the circular dependency.

Reply:
I'll go ahead and merge for now, but if there's any further feedback about this, do let me know here.  I'll still be working around this area of the code, and I'm open to making this better - I'm not fully married to my current solution.

One thing I've been asking myself when writing this code is around ownership.

Which class owns activating a tab?  Should it be `WPTabController` or the coordinators?

If we make a call on that I can shift the code accordingly and document our decision in the code.